package.json

```diff
@@ -5,18 +5,21 @@
     "packages/*"
   ],
   "scripts": {
+    "prepare": "yarn run build",
     "start": "yarn run storybook",
     "test": "jest",
     "test:size": "bundlesize",
     "lint": "eslint --ext .js,.ts,.tsx .",
     "storybook": "start-storybook --quiet --port 6006 --ci --static-dir .storybook/static",
     "storybook:build": "build-storybook --quiet --output-dir website/stories --static-dir .storybook/static",
-    "build:umd": "rollup --config",
-    "build:es": "babel src --root-mode upward --extensions '.js,.ts,.tsx' --ignore '**/__tests__/**/*','**/__mocks__/**/*','**/__fixtures__/**/*' --out-dir dist/es --quiet",
-    "build:prepare": "yarn run build:es",
-    "build:types": "tsc --declaration --declarationMap --outDir dist/es --emitDeclarationOnly",
-    "build": "yarn run build:umd && yarn run build:es && yarn run build:types",
-    "watch": "babel src --root-mode upward --extensions '.js,.ts,.tsx' --ignore '**/__tests__/**/*','**/__mocks__/**/*','**/__fixtures__/**/*' --out-dir dist/es --quiet --watch",
+    "build:clean": "lerna run build:clean --scope @francoischalifour/autocomplete-*",
+    "build:umd": "lerna run build:umd --scope @francoischalifour/autocomplete-*",
+    "build:esm": "lerna run build:esm --scope @francoischalifour/autocomplete-*",
+    "build:types": "lerna run build:types --scope @francoischalifour/autocomplete-*",
+    "build": "lerna run build --scope @francoischalifour/autocomplete-*",
+    "watch": "lerna run watch --parallel --scope @francoischalifour/autocomplete-*",
+    "dev": "yarn build && yarn watch-and-storybook",
+    "watch-and-storybook": "concurrently \"yarn run watch\" \"yarn run storybook\"",
```

Comment:
`yarn run watch & yarn run storybook`

This will run `yarn run watch` in the background and `yarn run storybook` in the foreground.

It means if you ctrl+c, it will only kill `yarn run storybook`, and `yarn run watch` will be still running in the background and you have to `kill -9 pid` to kill it.